test/hex_api_tests.erl

```diff
@@ -0,0 +1,22 @@
+-module(hex_api_tests).
+-include_lib("eunit/include/eunit.hrl").
+-define(OPTIONS, [
+    {client, #{adapter => hex_http_test, user_agent_string => "(test)"}},
+    {uri, "https://api.test"}
+]).
+% -define(OPTIONS, hex_api:default_options()).
+
+get_package_test() ->
+    {ok, Package} = hex_api:get_package("ecto", ?OPTIONS),
+    #{<<"name">> := <<"ecto">>, <<"releases">> := _} = Package,
```

Comment:
another option is to return atom keys so that it's easier to work with, mimics registry functions (also atom keys), and we could document all fields as a type. wdyt @ericmj?

somewhat related, should it be `hex_api:get_package("ecto")` or `hex_api:get_package(<<"ecto">>)` here and in other places. I'm starting to think the latter would be better because we almost always return binaries not charlists.

Reply:
Atom keys sound good. We should binaries over charlists as much as possible.